### ISSUE TITLE
Cannot access the links of the mobile navigation menu with the keyboard

### ISSUE THREAD

**Issue by Tanvez** [Harvard-University-iCommons/chainlit#29]
| IMPACT      | URL          | ISSUE DESCRIPTION   | WCAG CRITERIA | TESTING METHOD | SCREENSHOT or SNIPPET | ACTION |
| :----:          |    :----:      |      :----:                        |  :----:                    | :----:                        | :----:                                     |  :----:    | 
| High    | n/a        | Cannot access the links of the mobile navigation menu with the keyboard.  | 2.1.1 Keyboard - A | Manual - Chrome/Keyboard | [link to the screenshot](https://share.getcloudapp.com/ApugdNqx) | The issue may be that focus is managed here in a way that when focus is moved from the <div tabindex="0" class="css-l5c1s3"> element the menu closes before getting to the links. There is no need to put a tabfocus on this element.Try removing that tabfocus attribute and instead move focus to the first link in the navigation list.
